run_operations.py

```diff
@@ -96,7 +111,12 @@ def generate_templates_with_options(csv, dataset, sheet):
     return template_files
 
 def get_datasets_to_process(template_files, dataset=None):
-
+    """
+    Converts the template files being used in dataset/project names
```

Comment:
`template_files` is never used again. Its just to grab all of the dataset names, because I don't want to refer to things by their `template_file` name. I would prefer not to inline it if possible, as I don't see how inlining it, versus leaving it as a function, changes anything.